de/manual/ch_GUIMenus.xml

```diff
@@ -6658,535 +6658,1179 @@ url="&url-docs-de;guide/chapter_txns.html#txns-regstyle1"> &app; Kurs und Konzep
+
+          <para>Die im Menü &gm.bus; angezeigten Elemente sind die gleichen wie in
+            <xref linkend="gui-window-business-menu" />. Ergänzt wird das Menü mit den folgenden
+            Punkten.
+          </para>
+
+          <table pgwide="1" frame="topbot" id="gui-business-invoice-businessmenu">
+            <title>Geschäft - Geschäft-Menü - Offene-Posten-Verwaltung für kleine Unternehmen</title>
+
+            <titleabbrev>Geschäft - Geschäft-Menü</titleabbrev>
+
+            <tgroup align="left" cols="3">
+              <colspec colname="c1"></colspec>
+
+              <colspec colname="c2"></colspec>
+
+              <colspec colname="c3"></colspec>
+
+              <thead>
+                <row>
+                  <entry>
+                    <para>Menüpunkt
+                    </para>
+                  </entry>
+
+                  <entry align="center" namest="c2" nameend="c3">
+                    <para>Beschreibung
+                    </para>
+                  </entry>
+                </row>
+              </thead>
+
+              <tbody>
+                <row>
+                  <entry>
+                    <para>&gmi.bus.man-doc-link;
+                    </para>
+                  </entry>
+
+                  <entry namest="c2" nameend="c3">
+                    <para>Eine Verknüpfung eines Dokuments mit der Rechnung hinzufügen, bearbeiten oder Löschen.
+                    </para>
+                  </entry>
+                </row>
+
+                <row>
+                  <entry>
+                    <para>&gmi.bus.open-doc;
+                    </para>
+                  </entry>
+
+                  <entry namest="c2" nameend="c3">
+                    <para>Öffnet das mit der Rechnung verknüpfte Dokument.
+                    </para>
+
+                    <note>
+                      <para>Nur Verfügbar, wenn ein Dokument oder ein Speicherort mit der Rechnung verknüpft ist.
+                      </para>
+                    </note>
+                  </entry>
+                </row>
+
+                <row>
+                  <entry>
+                    <para>&gmi.bus.pay-inv;
+                    </para>
+
+                    <para>&gmi.bus.pay-vou;
+                    </para>
+                  </entry>
+
+                  <entry namest="c2" nameend="c3">
+                    <para>Öffnet den Dialog <guilabel>Zahlung verarbeiten</guilabel> mit dem die Rechnung gebucht werden
+                      kann.
+                    </para>
+                  </entry>
+                </row>
+              </tbody>
+            </tgroup>
+          </table>
+        </sect4>
+
+        <sect4 id="gui-business-invoice-report-menu">
+          <title>Geschäft - &gm.rep;-Menü</title>
+
+          <para>Die im Menü &gm.bus; angezeigten Elemente sind die gleichen wie in
+            <xref linkend="gui-window-reports-menu" /> sowie zusätzlich die folgenden Menüpunkte.
```

Comment:
z. Zt.:
`der folgende Menüpunkt.`
oder zukuntssicher nur
`<xref linkend="gui-window-reports-menu" />`<b>,</b> sowie zusätzlich
Vielleicht auch weniger holprig:
`Im Menü &gm.bus; gibt es zusätzlich zu den in <xref linkend="gui-window-reports-menu" /> aufgezählten Einträgen noch:`